cmd/eno-controller/main.go

```diff
@@ -3,31 +3,47 @@ package main
 import (
 	"flag"
 	"fmt"
+	"io"
 	"os"
 	"strings"
 	"time"
 
+	"github.com/go-logr/logr"
 	"github.com/go-logr/zapr"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
+	"k8s.io/client-go/rest"
 	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
 
+	v1 "github.com/Azure/eno/api/v1"
 	"github.com/Azure/eno/internal/controllers/aggregation"
 	"github.com/Azure/eno/internal/controllers/replication"
 	"github.com/Azure/eno/internal/controllers/rollout"
 	"github.com/Azure/eno/internal/controllers/synthesis"
 	"github.com/Azure/eno/internal/controllers/watchdog"
+	"github.com/Azure/eno/internal/execution"
 	"github.com/Azure/eno/internal/manager"
 )
 
 func main() {
-	if err := run(); err != nil {
+	if len(os.Args) > 1 && os.Args[1] == "install-executor" {
```

Comment:
The controller binary is reused for the "executor" process. Maybe we should release a separate image later, but it's so small I don't think it's worth the effort currently.